### PR TITLE
Fix IDB padding computation

### PR DESCRIPTION
**Pull request by guedou**
This PR fixes an issue with IDB padding computation.

@gpotter2 if you are OK with the fix, can we merge it today? I need it for a workshop.

### CODE REVIEW

File: scapy/utils.py [hunk 1]
```diff
@@ -1624,7 +1624,10 @@ def _read_block_dsb(self, block, size):
             raise EOFError
 
         # Compute the secrets length including the padding
-        padded_secrets_length = secrets_length + (4 - secrets_length % 4)
+        if secrets_length % 4:
+            padded_secrets_length = secrets_length + (4 - secrets_length % 4)
```
Comment (gpotter2) [inline scapy/utils.py:1628]: Can't you just do `secrets_length + (-4) % secrets_length` to work in all cases?

**Review by gpotter2 — CHANGES_REQUESTED** [review]

File: scapy/utils.py (hunk 1 above)
Comment (guedou) [reply scapy/utils.py:1628]: I am trying to compute the opposite, i.e. the padded length giving the secrets_length.

Comment (gpotter2) [reply scapy/utils.py:1628]: Yeah, `secrets_length + (-secrets_length) % 4` does that

Comment (guedou) [reply scapy/utils.py]: Wow. I am dumb, or tired, or both =/
Thanks for your fix.

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**